libbeat/beat/setup.go

```diff
@@ -0,0 +1,10 @@
+package beat
+
+type TemplateConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type TemplateConfig should have comment or be unexported